.github/workflows/python-format.yml

```diff
@@ -2,7 +2,7 @@ name: autoblack
 on: [pull_request]
 
 jobs:
-  autoblack:
+  setup:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repo
```

Comment:
You need to check out the code and run setup-python in each job, they can't run in a setup job as they are all running in their own containers.